packages/hub/readme.md

```diff
@@ -57,6 +57,20 @@ NODE_ENV=test yarn db:create
 yarn test:ci
 ```
 
+### Troubleshooting Test Errors
+
+If you run into the following error
+
+```shell
+    getaddrinfo ENOTFOUND RequiredInLocalDotenv RequiredInLocalDotenv:5432
+```
+
+Make sure you have a local .env file dedicated to testing, by running the following:
+
+```shell
+cp .env.development.local .env.test.local
```

Comment:
There might be folks who run tests locally before setting up the development environment. In that case, `.env.development.local` would not exist. Would it be better to state something along the lines of:

The contents of your `.env.test.local` file define the connection to the postgres db via `HUB_DB_HOST` and `HUB_DB_PORT` variables. An example is:
```
HUB_DB_HOST=localhost
HUB_DB_PORT=5432
```

Reply:
Actually, with https://github.com/statechannels/monorepo/pull/695, we can tell folks that `.env.test.local` is only needed if defaults in `.env.test` need to be overridden.

Reply:
Hmm so as of #693, should we just close this (and #697) then?

Reply:
@kerzhner do you want to just include another commit in #695 then to update the README for the above comment? (only needing `.env.test.local` in case defaults need to be overridden)